.github/workflows/ci.yml

```diff
@@ -68,6 +68,13 @@ jobs:
         run: |
           ./meta-dts/scripts/generate-ipxe-menu.sh ${{steps.dts-ver.outputs.DTS_VER}}
           scp -i ~/.ssh/dts-ci-key dts.ipxe builder@10.1.40.2:boot/dts/
+      - name: Trigger signing:
+        shell: bash
+        run: |
+          DTS_VER="${{steps.dts-ver.outputs.DTS_VER}}"
+          git clone https://git.3mdeb.com/3mdeb/dts-release-cicd-pipeline.git
+          git tag `echo $DTS_VER | tr -d "v"`
+          git push origin main --tags
```

Comment:
Trigger automatic signing via pushing tag with actual version to a repo which is located on self-hosted git instance